### PR TITLE
Change secret name to digitalocean

### LINKED ISSUE #20: Align k8s digitalocean token secret name with cloud-controller-manager
There is a difference in the name between how digitalocean/cloud-controller-manager calls the secret that holds the k8s digitalocean token, and the way the secret is named here. Leading to two secrets with different names in k8s.

Is it possible to align these ?

### PR DESCRIPTION
**Pull request by fatih**
This means that users that already have created a token for
cloud-controller-manager can use the same token for the CSI plugin

- [x] checkout namespace compatibility

Fixes #20

### CODE REVIEW

**Comment by fatih** [comment]
@andrewsykim can you please comment on the namespace compatibility. I see that CCM is using `kube-system`, but we don't do it for CSI. I believe therefore, even if I change the name, it's not going to work out because the CCM secret lives in a separate namespace. The CSI secret lives in the `default` namespace. Happy to change it for our own case, just wonder the reason why CCM uses `kube-system`.

File: deploy/kubernetes/releases/csi-digitalocean-NOT_RELEASED_v0.0.2.yaml [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+            - name: ADDRESS
+              value: /csi/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /csi/
+              # TODO(arslan): the registar is not implemented yet
```
Comment (aybabtme) [inline deploy/kubernetes/releases/csi-digitalocean-NOT_RELEASED_v0.0.2.yaml]: `registrar`

**Comment by andrewsykim** [comment]
CCM uses `kube-system` since it is technically a component of the kubernetes control plane. Like you said I think the best option going forward (at least for now) is just to duplicate the secret for this use case.

**Comment by fatih** [comment]
I see. I'm not sure why the CSI plugin should live in `kube-system`. That namespace ism meant for services that are [created by Kubernetes](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/):

> The namespace for objects created by the Kubernetes system

Do you think it's better we change the default namespace for CCM to `default`?

**Comment by aybabtme** [comment]
Seems to me that if `CCM` is a `kube-system` then the case is even stronger for `CSI` to be a `kube-system`?

**Comment by fatih** [comment]
@aybabtme true that. Moving it to `kube-system`. Will update the PR now

**Comment by fatih** [comment]
@aybabtme @andrewsykim moved to `kube-system` and fixed review comments. PTAL.

**Comment by fatih** [comment]
@lxfontes thanks for catching these. Fixed it.

**Comment by lxfontes** [comment]
lgtm

**Comment by cagedmantis** [comment]
lgtm

**Pull request merged**